examples/brands.tf

```diff
@@ -0,0 +1,20 @@
+# brands.tf
+#   Brand example
+#
+# API reference:
+#   https://developer.zendesk.com/rest_api/docs/support/brands
+
+resource "zendesk_brand" "T-800" {
```

Comment:
@tamccall 
I have a question. Can we get other type of resource data from `CreateBrand()`?
I want to set attachment for brand logo like this,

```tf
resource "zendesk_brand" "T-800" {
  name            = "T-800"
  active          = true
  has_help_center = true
  subdomain       = "d3v-terraform-provider-t800"

  logo_attachment_id = "${zendesk_attachment.street.id}"
}
```
schema: https://github.com/nukosuke/terraform-provider-zendesk/pull/86/files#diff-ab422a881b6d29eb284acd18b20e7c6cR58

But, Zendesk API requires the full fields not only attachment id.
https://developer.zendesk.com/rest_api/docs/support/brands#example

I could come up with getting attachment data from API  by `zendesk.GetAttachment`.
But, it does not make sense because tfstate already has attachment data then.

Let me know if you have any idea.

Reply:
ahh yeah that is unfortunate.

i think we are going to have to go with the API approach.

I don't really foresee any issues with us making multiple API calls in the Create/Update functions, and even if we can read find a way to read the attachment out of the terraform state we will need to account for cases where the attachment was not created using terraform.

Reply:
Thank you for your advise.
Finally, I couldn't find the way to get attachment data from `createBrand` and decided to take API approach.

> we will need to account for cases where the attachment was not created using terraform

It was a blind spot in my mind. Thanks.